# API总是出现如下错误

HTTPConnectionPool(host='dataapi01.cloud-data.datayes.com', port=80): Request timed out. (timeout=30)